### PR TITLE
Add floating ip sample to rdns documentation

### LINKED ISSUE #33: Add example to documentation how to set reverse dns for a floating ip
tried it by setting `ip_address` to the server ip address and specifing the `floating_ip_id`. this works but only set the reverse dns for the server and not for the floating ip. 
then i tried to use the floating ip address as `ip_address` but then i get the following error message: `hcloud_rdns.master_float: IP does not belong to the server (forbidden)`

```
resource "hcloud_floating_ip" "main_ip" {
  type = "ipv4"
  home_location = "nbg1"
}

//this works but only sets the rdns for the server
resource "hcloud_rdns" "master" {
  server_id = "${hcloud_server.test01.id}"
  ip_address = "${hcloud_server.test01.ipv4_address}"
  floating_ip_id = "${hcloud_floating_ip.main_ip.id}"
  dns_ptr = "example.com"
}

//this produces the error stated above
resource "hcloud_rdns" "master_float" {
  server_id = "${hcloud_server.test01.id}"
  ip_address = "${hcloud_floating_ip.main_ip.ip_address}"
  floating_ip_id = "${hcloud_floating_ip.main_ip.id}"
  dns_ptr = "example.com"
}
```

### PR DESCRIPTION
**Pull request by LKaemmerling**
Close #33

### CODE REVIEW

**Comment by LKaemmerling** [comment]
@thcyron

**Comment by c33s** [comment]
what about a more complex example?
```
resource "hcloud_floating_ip" "floating1" {
  type = "ipv4"
  home_location = "nbg1"
}

resource "hcloud_rdns" "floating_master" {
  ip_address = "${hcloud_floating_ip.floating1.ip_address}"
  floating_ip_id = "${hcloud_floating_ip.floating1.id}"
  dns_ptr = "example.com"
}

resource "hcloud_floating_ip_assignment" "floating1_assignment" {
  floating_ip_id = "${hcloud_floating_ip.floating1.id}"
  server_id = "${hcloud_server.myserver.id}"
}
```
mainly the part from your comment https://github.com/terraform-providers/terraform-provider-hcloud/issues/33#issuecomment-423947740 to show the use with `rdns`, `ip_address` & `floating_ip_id`

File: website/docs/r/rdns.html.md [hunk 1]
```diff
@@ -26,6 +27,19 @@ resource "hcloud_rdns" "master" {
...
 
+For Floating IPs:
+```hcl
+resource "hcloud_floating_ip" "floating1" {
+  home_location = "ngb1"
+  type = "ipv4"
+}
+
+resource "hcloud_rdns" "floating_master" {
+  server_id = "${hcloud_floating_ip.floating1.id}"
```
Comment (c33s) [inline website/docs/r/rdns.html.md]: i don't understand why `server_id` is `floating.id` here. shouldn't it map to a server id or shouldn't we use `floating_ip_id` here?

Comment (LKaemmerling) [reply website/docs/r/rdns.html.md]: Sorry this was just a typo. I have fixed it.

**Pull request merged**